pkg/controller/clone-controller.go

```diff
@@ -37,7 +38,7 @@ func NewCloneController(client kubernetes.Interface,
 	pullPolicy string,
 	verbose string) *CloneController {
 	c := &CloneController{
-		Controller: *NewController(client, pvcInformer, podInformer, image, pullPolicy, verbose),
+		Controller: *NewController(client, pvcInformer, podInformer, image, pullPolicy, verbose, "cloner"),
```

Comment:
We should probably create a variable set for these, in Controller you have "ctlType string" that works, you could just add a const in each of these controllers (or even a variable would be better than free form string) to define each of the controller types strings, and make them something easily referenced throughout the code.

Another possible suggestion would be to define an enum in Controller.go and use that but that's probably overkill here I think.